include/mt32emu/c_interface/meson.build

```diff
@@ -1,3 +1,6 @@
-foreach file_name, src_file : mt32emu_c_header_map
-  configure_file(input : src_file, output : file_name, copy : true)
+foreach src_file : c_headers
+  file_name = 'string : @0@'.format(src_file).split('/')
```

Comment:
What is this `string : ` syntax? Is it documented somewhere?

Reply:
https://mesonbuild.com/Syntax.html#string-formatting
and now that I re-read it, only need:
```
file_name = '@0@'.format(src_file).split('/')
```
I did initially try *src_file.split('/')* , but it wasn't happy, hence the format